src/game_inventory.cpp

```diff
@@ -1,21 +1,53 @@
 #include "game.h"
 
+#include "cata_utility.h"
 #include "inventory_ui.h"
 #include "player.h"
 #include "item.h"
 #include "itype.h"
 
+item_location inv_internal( player &p, const inventory_selector_preset &preset,
+                            const std::string &title, int radius,
+                            const std::string &none_message )
+{
+    p.inv.restack( &p );
+    p.inv.sort();
+
+    inventory_pick_selector inv_s( p, preset );
+
+    inv_s.set_title( title );
```

Comment:
I would avoid adding unnecessary constraints and passing more arguments to ancestor constructors unless there's a valid reason to do so. Also:
- The title might depend upon selector's content (which can be revealed only after construction)
- I have `set_hint()` function which does similar thing and must be set **after** construction because it already depends on the content. I want consistency here
- This concept is likely to change and more (or less) optional arguments (properties in terms of C#) can persist in the class, but don't worth extracting yet another. Passing all of them on construction would require rewriting all of them each time some of them change.

Conclusion: I won't do that.